### ISSUE TITLE
Is the example in the new documentation executable?

### ISSUE THREAD

**Issue by dwoznicki** [aws/aws-sdk-js-v3#4685]
### Describe the issue

That is, can I run the code snippet in the browser and see either a dummy response, or a response from my actual AWS configuration?* If so, how can it be run? If not, why is the snippet editable in the browser?

\* For what it's worth, I'd prefer a dummy response. Generally, I do not want to have to sign in to use documentation.

### Links

https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/preview/client/api-gateway/command/GetRestApisCommand/

**Comment by RanVaknin** [comment]
Hi @dwoznicki ,

Thanks for reaching out. Im really confused as to what seems to be the problem.

Can you please paste the actual "dummy" response you are getting, paste your actual code. Tell us exactly how you are running your code in the browser? are you bundling it with webpack and then running it on the browser? 

>* For what it's worth, I'd prefer a dummy response. Generally, I do not want to have to sign in to use documentation.

I'm not sure I understand. You do not need to sign in to use documentation.

Thanks,
Ran~

**Comment by dwoznicki** [comment]
Sorry, I wasn't clear.

I opened this issue because of the user experience I had visiting [this page](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/preview/client/api-gateway/command/GetRestApisCommand/) for the first time. Here's how it went.

- I opened the page, and immediately found the layout much easier to read than the old UI. :+1: 
- I scrolled down the page, looking for documentation on the output of this API endpoint, but couldn't find anything. This surprised me, so I started looking for other ways to check the expected output.
- I noticed that the example syntax is editable.
  ![image](https://user-images.githubusercontent.com/13933454/235768159-d1f12bb9-3255-49f6-bf61-e9e186bbd31f.png)
  Some sites have playgrounds for their documentation, where you can edit + execute the code in browser. I thought maybe that was the case here, so I played around trying to execute the code and get a response.
- I was unable to find a way to execute the code snippet, presumably because it is not meant to be executed.

Basically, I think there is a combination of two problems that lead me to open this issue.

1. The docs don't show the expected output of an API endpoint.
2. The example syntax is editable. Why am I allowed to edit these snippets in the browser?

**Comment by MYoung25** [comment]
Hi @dwoznicki,

I hope I can clarify a few things, but first, **thank you** for taking time to leave us feedback!

1. We are working on adding output shapes to the Example Syntax section
2. You are also not the only person confused by that section being editable, this is our fault and we are aware of it.

**Comment by MYoung25** [comment]
I'm happy to share that the output shapes are now showing in the Example Syntax sections!
